src/styles/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-internal-gap;
+  padding: 200px $calendar-outer-gap 0;
+  max-width: $calendar-width;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $cell-width;
+    width: $cell-width;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+
+    background-color: #eee;
+    border: 1px solid #000;
+
+    transition: ease 0.5s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated